src/core/server/plugins/plugins_system.ts

```diff
@@ -262,6 +262,7 @@ export class PluginsSystem<T extends PluginType> {
               uiPluginNames.includes(p)
             ),
             requiredBundles: plugin.manifest.requiredBundles,
+            enableForAnonymousPages: plugin.manifest.enableForAnonymousPages,
```

Comment:
ultra-NIT: for consistency with the form of `requiredBundles`, maybe `enabledOnAnonymousPages`

Reply:
Fixed in b638b927c8e81422517efde2dd8a7f2808b6716b.